### PR TITLE
Use ellipses in Mechanics

### PR DESCRIPTION
**Pull request by tobolar**
Use ellipses (a row of three points) in documentation of MultiBody, Rotational and Translational.

refs #2749

### CODE REVIEW

File: Modelica/Mechanics/Rotational.mo [hunk 1]
```diff
@@ -3683,8 +3683,8 @@ distributions:
    Typical values of coefficients of friction:
 </p>
 <pre>
-      dry operation   :  <strong>mue</strong> = 0.2 .. 0.4
-      operating in oil:  <strong>mue</strong> = 0.05 .. 0.1
+      dry operation   :  <strong>mue</strong> = 0.2 ... 0.4
+      operating in oil:  <strong>mue</strong> = 0.05 ... 0.1
 </pre>
```
Comment (beutlich) [inline Modelica/Mechanics/Rotational.mo]: Why not here?

File: Modelica/Mechanics/Rotational.mo [hunk 2]
```diff
@@ -3840,8 +3840,8 @@ distributions:
    Typical values of coefficients of friction:
 </p>
 <pre>
-      dry operation   :  <strong>mue</strong> = 0.2 .. 0.4
-      operating in oil:  <strong>mue</strong> = 0.05 .. 0.1
+      dry operation   :  <strong>mue</strong> = 0.2 ... 0.4
+      operating in oil:  <strong>mue</strong> = 0.05 ... 0.1
```
Comment (beutlich) [inline Modelica/Mechanics/Rotational.mo]: Why not here?

File: Modelica/Mechanics/Rotational.mo [hunk 3]
```diff
@@ -4045,8 +4045,8 @@ Typical values of coefficients of friction:
 </p>
 
 <blockquote><pre>
-dry operation   :  <strong>mue</strong> = 0.2 .. 0.4
-operating in oil:  <strong>mue</strong> = 0.05 .. 0.1
+dry operation   :  <strong>mue</strong> = 0.2 ... 0.4
+operating in oil:  <strong>mue</strong> = 0.05 ... 0.1
```
Comment (beutlich) [inline Modelica/Mechanics/Rotational.mo]: Why not here?

File: Modelica/Mechanics/Translational.mo [hunk 4]
```diff
@@ -3146,8 +3146,8 @@ distributions:
    Typical values of coefficients of friction:
 </p>
 <pre>
-      dry operation   :  <strong>mue</strong> = 0.2 .. 0.4
-      operating in oil:  <strong>mue</strong> = 0.05 .. 0.1
+      dry operation   :  <strong>mue</strong> = 0.2 ... 0.4
+      operating in oil:  <strong>mue</strong> = 0.05 ... 0.1
```
Comment (beutlich) [inline Modelica/Mechanics/Translational.mo]: Why not here?

File: Modelica/Mechanics/Translational.mo [hunk 5]
```diff
@@ -1861,14 +1861,14 @@ force law in a target system between two masses.
         end for;
         annotation (Documentation(info="<html>
 <p>
-Returns a table with the friction characteristic table[nTable,2] = [0, f1; ....; v_max, fn], where the first
-column is the velocity v in the range 0..v_max and the second column is the friction force
+Returns a table with the friction characteristic table[nTable,2] = [0,&nbsp;f1; &hellip;; v_max,&nbsp;fn], where the first
```
Comment (beutlich) [inline Modelica/Mechanics/Translational.mo]: Why no nbsp here before and after the ellipsis?

File: Modelica/Mechanics/MultiBody/Visualizers.mo [hunk 6]
```diff
@@ -2121,8 +2121,8 @@ Additionally, <strong>external shapes</strong> can be specified as (not all opti
 </p>
 
 <ul>
-<li> <strong>\"1\", \"2\", ...</strong><br>
-     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", ...
+<li> <strong>\"1\", \"2\", &hellip;</strong><br>
+     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", &hellip;
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Visualizers.mo]: Why no nbsp here before the ellipsis?

File: Modelica/Mechanics/Translational.mo (hunk 5 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational.mo]: Well then this should be:  `table[nTable,2]&nbsp;=&nbsp;[0,&nbsp;f1;&nbsp;&hellip;;&nbsp;v_max,&nbsp;fn]` 
But then the non-breaking text would be too long to my feeling.

Comment (tobolar) [reply Modelica/Mechanics/Translational.mo]: I have changed this.

File: Modelica/Mechanics/MultiBody/Visualizers.mo (hunk 6 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: OK

**Comment by tobolar** [comment]
@beutlich I have just incorporated your comments. Please review again.

File: Modelica/Mechanics/Translational.mo [hunk 7]
```diff
@@ -1861,14 +1861,15 @@ force law in a target system between two masses.
...
+and the second column is the friction force according to the Stribeck curve:
 </p>
-<pre>
-  F_Coulomb + F_prop*v + F_Stribeck*exp(-fexp*v);
-</pre>
 
+<blockquote><pre>
+f = F_Coulomb + F_prop*v + F_Stribeck*exp(-fexp*v);
+</pre></blockquote>
 </html>"));
```
Comment (beutlich) [inline Modelica/Mechanics/Translational.mo]: pre needed here?

Comment (dietmarw) [reply Modelica/Mechanics/Translational.mo]: Example codes that refer to existing parameter/variable names should in my opinion always set in `<code>...</code>` or `<pre>`.

**Comment by tobolar** [comment]
@dietmarw, @beutlich Ok to close this PR?

**Comment by beutlich** [comment]
Closing /w or w/o merge you mean?

**Comment by tobolar** [comment]
Sorry, I mean to merge of course.

File: Modelica/Mechanics/Translational.mo (hunk 4 above)
Comment (tobolar) [reply Modelica/Mechanics/Translational.mo]: Fixed in c50a207b16525be5da129ec427f6a3c75687d173

**Comment by tobolar** [comment]
@dietmarw Fixed in 86624751adfb7b344dc21f6b005a8f6e688484b3, thanks.

**Comment by beutlich** [comment]
Merge conflicts are resolved now.

File: Modelica/Mechanics/MultiBody/Types.mo [hunk 8]
```diff
@@ -153,8 +153,8 @@ Additionally, external shapes can be specified as (not all options might be supp
 </p>
 
 <ul>
-<li> <strong>\"1\", \"2\", ...</strong><br>
-     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", ...
+<li> <strong>\"1\", \"2\",&nbsp;&hellip;</strong><br>
+     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", &hellip;
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Types.mo]:
```suggestion
     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\",&nbsp;&hellip;
```

File: Modelica/Mechanics/MultiBody/Visualizers.mo [hunk 9]
```diff
@@ -82,8 +82,8 @@ Additionally, <strong>external shapes</strong> can be specified as (not all opti
 </p>
 
 <ul>
-<li> <strong>\"1\", \"2\", ...</strong><br>
-     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", ...
+<li> <strong>\"1\", \"2\",&nbsp;&hellip;</strong><br>
+     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\", &hellip;
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Visualizers.mo]:
```suggestion
     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\",&nbsp;&hellip;
```

File: Modelica/Mechanics/MultiBody/Visualizers.mo [hunk 10]
```diff
@@ -321,8 +321,8 @@ Additionally, <strong>external shapes</strong> can be specified as (not all opti
...
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Visualizers.mo]:
```suggestion
     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\",&nbsp;&hellip;
```

File: Modelica/Mechanics/MultiBody/Visualizers.mo [hunk 11]
```diff
@@ -2121,8 +2121,8 @@ Additionally, <strong>external shapes</strong> can be specified as (not all opti
...
```
Comment (beutlich) [inline Modelica/Mechanics/MultiBody/Visualizers.mo]:
```suggestion
     define external shapes specified in DXF format in files \"1.dxf\", \"2.dxf\",&nbsp;&hellip;
```

File: Modelica/Mechanics/MultiBody/Types.mo (hunk 8 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Types.mo]: Fine, thanks.

File: Modelica/Mechanics/MultiBody/Visualizers.mo (hunk 9 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: Fine, thanks.

File: Modelica/Mechanics/MultiBody/Visualizers.mo (hunk 10 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: Fine, thanks.

File: Modelica/Mechanics/MultiBody/Visualizers.mo (hunk 11 above)
Comment (tobolar) [reply Modelica/Mechanics/MultiBody/Visualizers.mo]: Fine, thanks.

**Pull request merged**